### PR TITLE
fix(#172): Remove reference to obsoleted `writeStatic`

### PR DESCRIPTION
**Pull request by pham**
# Summary

Fixes broken build with breaking change introduced in SvelteKit [PR5618](https://github.com/sveltejs/kit/pull/5618).

Fixes: #172

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "serverless",
     "ssr"
   ],
-  "version": "0.14.1",
```
Comment (jthegedus) [inline package.json:24]: You shouldn't change this or the Changelog manually, it will be done by automation during release.

File: src/index.js [hunk 2]
```diff
@@ -95,7 +95,6 @@ const entrypoint = function (options = {}) {
 			builder.rimraf(publicDir);
 
 			builder.log.minor(logRelativeDir('Writing client application to', publicDir));
-			builder.writeStatic(publicDir);
```
Comment (jthegedus) [inline src/index.js:98]: Is this replaced with anything? How are these files now being written?

Comment (sspickle) [reply src/index.js:98]: Hmm... I'm just experimenting with the adapter, but stuck on this error. Any guidance? Just wait for this to be resolved? pin to an earlier version of SvelteKit? Thoughts? Thanks.

Comment (Nushio) [reply src/index.js:98]: It doesn't look like it was replaced with anything else on the adapter-vercel
https://github.com/sveltejs/kit/commit/819172065cdda6c695f3fb48e60fa451c1caaae6

I'll give this a try.

Comment (pham) [reply src/index.js:98]: According to [PR5618](https://github.com/sveltejs/kit/pull/5618):
> Have writeClient write both the client and static files and remove writeStatic. This is a breaking change for adapter authors.

`writeClient` does job for `writeStatic` so I removed `writeStatic`.

**Review by jthegedus — APPROVED** [review]

**Review by akash10008 — APPROVED** [review]

**Pull request merged**